custom_components/auth_oidc/__init__.py

```diff
@@ -50,17 +51,19 @@ async def async_setup(hass: HomeAssistant, config):
 
     _LOGGER.info("Registered OIDC provider")
 
-    # We only use openid, profile & groups, never email
-    scope = "openid profile groups"
```

Comment:
You probably based your PR of the wrong branch, can you rebase?

Reply:
Seems like this change is still there, any reason why you need this scope removed (maybe it doesn't work with Keycloak)?

If you need that, we should make a config flag to set the scopes instead which defaults to openid email groups.

Reply:
Comment still changed.